pyproject.toml

```diff
@@ -4,8 +4,10 @@ version = "0.1.0"
 description = "Calibrates a residential building energy model to utility bills."
 readme = "README.md"
 authors = [
-    { name = "Noel Merket", email = "noel.merket@nrel.gov" },
     { name = "Nathan Moore", email = "nathan.moore@nrel.gov" },
+    { name = "Benjamin Park", email = "benjamin.park@nrel.gov" },
+    { name = "Scott Horowitz", email = "scott.horowitz@nrel.gov" },
+    { name = "Noel Merket", email = "noel.merket@nrel.gov" },
```

Comment:
@nmerket Should this retain your NREL email address or be updated to your Pearl address?

Reply:
Update to my pearl address, I suppose. Thanks for asking.